examples/jsm/loaders/DRACOLoader.js

```diff
@@ -82,13 +82,18 @@ class DRACOLoader extends Loader {
 
 	}
 
+
 	parse( buffer, onLoad, onError ) {
 
-		this.decodeDracoFile( buffer, onLoad, null, null, SRGBColorSpace ).catch( onError );
+		this.decodeDracoFile( buffer, onLoad, null, null, SRGBColorSpace ).catch( ( error ) => {
```

Comment:
Can we add a default callback parameter to `parse()` so we can simply write `catch( onError );`? Maybe:
```js
parse( buffer, onLoad, onError = () => {} ) {
```
My hope is that would clean the code a bit more up. Would this also work for `decodeDracoFile()`?

Reply:
https://github.com/mrdoob/three.js/pull/27306/commits/14ddcc1779248e0983ea1a846dbfdee48fa25ee1

Reply:
Awesome!